aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/ReadHandler.java

```diff
@@ -0,0 +1,36 @@
+package software.amazon.rds.dbinstance;
+
+import java.util.List;
+
+import software.amazon.awssdk.services.rds.RdsClient;
+import software.amazon.awssdk.services.rds.model.DBInstance;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class ReadHandler extends BaseHandlerStd {
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+            final AmazonWebServicesClientProxy proxy,
+            final ResourceHandlerRequest<ResourceModel> request,
+            final CallbackContext callbackContext,
+            final ProxyClient<RdsClient> proxyClient,
+            final Logger logger) {
+
+        return proxy.initiate("rds::describe-db-instance", proxyClient, request.getDesiredResourceState(), callbackContext)
+                .translateToServiceRequest(Translator::describeDbInstancesRequest)
+                .makeServiceCall((describeRequest, proxyInvocation) -> proxyInvocation.injectCredentialsAndInvokeV2(
+                        describeRequest,
+                        proxyInvocation.client()::describeDBInstances
+                ))
+                .done((describeRequest, describeResponse, proxyInvocation, model, context) -> {
+                    final DBInstance dbInstance = describeResponse.dbInstances().stream().findFirst().get();
+                    final List<Tag> tags = listTagsSoftFailOnAccessDenied(proxyInvocation, dbInstance.dbInstanceArn());
```

Comment:
Sorry for late comment. describe response already have tag list in it.
Could you elaborate why we made anther `ListTagForResource` request instead of fetching them from response?
i.e: `dbInstance.tagList()`

Reply:
Good catch, thanks very much!